### PR TITLE
Events

### PR DESCRIPTION
**Pull request by fatima-najafi**
In this PR i implement these tasks:
- [x] Create a new directory called logic inside project
- [x] Copy the files calculate.js and operate.js into logic/ directory
- [x] Install big.js using NPM
- [x] Import calculate.js in  Calculator.js component
- [x] Implement click event handlers using the math logic from calculate.js and operate.js in component

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (Whoistolu) [inline src/logic/operate.js]: - [ ] [REQUIRED] You have done a great job @fatima-najafi , however, we need to remove the eslint disable in order to allow elisnt to standardize our JS code. Let's resolve the issue manually.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @fatima-najafi 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Highlight
🔆 The project has the expected `operate.js` and `calculate.js`
🔆 The `Calculator` component imports the `calculate.js` file from the logic directory
🔆 The `Calculator` app handles state with `useState`
🔆 The application works as expected

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .eslintrc.json [hunk 2]
```diff
@@ -1,3 +1,39 @@
...
+//   "overrides": [
+//     {
+//       // feel free to replace with your preferred file pattern - eg. 'src/**/*Slice.js' or 'redux/**/*Slice.js'
+//       "files": ["src/**/*Slice.js"],
+//       // avoid state param assignment
+//       "rules": { "no-param-reassign": ["error", { "props": false }] }
+//     }
+//   ],
+//   "ignorePatterns": ["dist/", "build/"]
+// }
```
Comment (Munsa1) [inline .eslintrc.json:36]: - [ ] (optional) consider removing commented settings, for a more clean codebase

**Review by Munsa1 — APPROVED** [review]
Hi @fatima-najafi, 
congrats

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Made Requested Changes By Previous Reviewer :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**